adapters/ddf-adapter/src/test/java/com/connexta/replication/adapters/ddf/CqlBuilderTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.adapters.ddf;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+
+import java.util.Date;
+import org.junit.Test;
+
+public class CqlBuilderTest {
+
+  @Test
+  public void equalTo() {
+    assertThat(CqlBuilder.equalTo("att", "val"), is("[ att = 'val' ]"));
+    assertThat(CqlBuilder.equalTo("att.sub", "val"), is("[ \"att.sub\" = 'val' ]"));
```

Comment:
❓ This tests escaping attribute names containing `.`. Should we have separate tests for `-` and `id` as well?

Reply:
Added